Meta/ladybird.py

```diff
@@ -394,7 +394,7 @@ def build_main(build_dir: Path, target: str | None = None, args: list[str] = [])
     try:
         subprocess.check_call(build_args)
     except subprocess.CalledProcessError as e:
-        print_process_stderr(e, f"Unable to build Ladybird {f"target {target}" if target else "project"}")
+        print_process_stderr(e, f"Unable to build Ladybird " + (f"target {target}" if target else "project"))
```

Comment:
Instead of concatenating strings, can't we just use single-quotes inside the f-string?
```python
print_process_stderr(e, f"Unable to build Ladybird {f'target {target}' if target else 'project'}")
```